assets/hashes.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should this file be under source control? If so, maybe at least add comments to explain what it's for...

Also, missing newline at EOF.

Reply:
Yes, it should be ignored.